package.json

```diff
@@ -16,7 +16,7 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.3.5",
```

Comment:
Please do not add useless files to your PR.
You can use **git add src/** command to add needed files
**git status** command — for checking file status